test/Tester/SerializationTests/ProtobufSerializationTests/ProtobufSerializationTests.cs

```diff
@@ -10,10 +10,13 @@ namespace UnitTests.Serialization
 {
     public class ProtobufSerializationTests
     {
+#if !NETSTANDARD_TODO
```

Comment:
Should this #if be around the whole class if Protobuf doesn't support .NET Core yet?